server/src/main.rs

```diff
@@ -21,7 +21,7 @@ mod schema;
 
 extern crate tracing;
 
-pub(crate) const MINA_GOVERNANCE_SERVER: &str = "mina_governance_server";
+pub(crate) const log_target: &str = "mina_on_chain_voting_api";
```

Comment:
This should be capitalized. https://buildkite.com/granola/mina-on-chain-voting/builds/116#018c6924-8e99-4a09-8f3b-3e7db04031ef/19-193

Also this should be a String rather than a str slice since it's a constant.